pkg/controller/servicemesh/controlplane/pruner.go

```diff
@@ -56,6 +56,7 @@ var (
 		schema.GroupVersionKind{Group: "networking.istio.io", Version: "v1alpha3", Kind: "Gateway"},
 		schema.GroupVersionKind{Group: "networking.istio.io", Version: "v1alpha3", Kind: "VirtualService"},
 		schema.GroupVersionKind{Group: "jaegertracing.io", Version: "v1", Kind: "jaeger"},
+		schema.GroupVersionKind{Group: "authentication.maistra.io", Version: "v1", Kind: "ServiceMeshPolicy"},
```

Comment:
How come there's no entry for `rbac.maistra.io/ServiceMeshRbacConfig` ?

Reply:
There was no entry for rbac.istio.io/ClusterRbacConfig

Reply:
One of my concerns when I first saw the code was the fragility, it would be worth revisiting this area at some point as I'm sure things will get out of sync quickly

Reply:
Doesn't `ClusterRbacConfig` need to be deleted when deleting a `SMCP`?

Reply:
`ClusterRbacConfig` shouldn't be used any more however if it should have been included before this change then we should create another issue to look into keeping these consistent.  For now I am just updating what already exists to support the new resources.

Reply:
OK. As far I understand, the (Cluster/ServiceMesh)RbacConfig _does_ need to be deleted. I'll file a Jira if you haven't already.

Reply:
I haven't created an issue for this yet, I was intending to add one covering the fragility of the approach.  Please add one.

Reply:
Jira issue: https://issues.jboss.org/browse/MAISTRA-729